pootle/apps/pootle_store/utils.py

```diff
@@ -0,0 +1,102 @@
+from pootle_comment.forms import UnsecuredCommentForm
+
+
+class SuggestionsReview(object):
+    accept_email_template = 'editor/email/suggestions_accepted_with_comment.txt'
+    accept_email_subject = _(u"Suggestion accepted with comment")
+    reject_email_template = 'editor/email/suggestions_rejected_with_comment.txt'
+    reject_email_subject = _(u"Suggestion rejected with comment")
+
+    def __init__(self, suggestions, reviewer):
+        self.suggestions = suggestions
+        self.reviewer = reviewer
+
+    def should_notify(self, comment):
+        return (
+            comment
+            and settings.POOTLE_EMAIL_FEEDBACK_ENABLED)
+
+    def add_comments(self, rejected=True, comment=""):
+        for suggestion in self.suggestions:
+            UnsecuredCommentForm(
+                suggestion,
+                dict(comment=comment,
+                     user=self.reviewer)).save()
+
+    def accept_suggestions(self):
+        for suggestion in self.suggestions:
+            suggestion.unit.accept_suggestion(
+                suggestion,
+                suggestion.unit.store.translation_project,
+                self.reviewer)
+
+    def accept(self, comment=""):
+        self.accept_suggestions()
+        if self.should_notify(comment):
+            self.notify_suggesters(rejected=False, comment=comment)
+        if comment:
+            self.add_comments(rejected=False, comment=comment)
+
+    def build_absolute_uri(self, url):
+        return site.get().build_absolute_uri(url)
+
+    def get_email_message(self, suggestions, comment, template):
+        for suggestion in suggestions:
+            suggestion.unit_url = (
+                self.build_absolute_uri(
+                    suggestion.unit.get_translate_url()))
+        return loader.render_to_string(
+            template,
+            dict(suggestions=suggestions,
+                 comment=comment))
+
+    def notify_accepted(self, suggester, suggestions, comment):
+        send_mail(
+            self.accept_email_subject,
+            self.get_email_message(
+                suggestions,
+                comment,
+                self.accept_email_template),
+            from_email=None,
+            recipient_list=[suggester.email],
+            fail_silently=True)
+
+    def notify_rejected(self, suggester, suggestions, comment):
+        send_mail(
+            self.reject_email_subject,
+            self.get_email_message(
+                suggestions,
+                comment,
+                self.reject_email_template),
+            from_email=None,
+            recipient_list=[suggester.email],
+            fail_silently=True)
+
+    def notify_suggesters(self, rejected=True, comment=""):
+        for suggester, suggestions in self.users_and_suggestions.items():
+            if rejected:
+                self.notify_rejected(suggester, suggestions, comment)
+            else:
+                self.notify_accepted(suggester, suggestions, comment)
```

Comment:
`notify_accepted` is basically repeating `notify_rejected` but changing the subject and the template. Perhaps it is better to get the subject and template based on the `rejected` value and have a single method to send email.